sirepo/api_auth.py

```diff
@@ -42,6 +43,8 @@ def check_api_call(qcall, func):
             qcall.auth.require_email_user()
         elif expect == a.REQUIRE_ADM:
             qcall.auth.require_adm()
+        elif expect == a.REQUIRE_NOT_DISABLED:
```

Comment:
REQUIRE_DISABLED will not be an expected by an API. It's a role that is checked in a way similar to the sym_type roles. It's more of an attribute-based access than a role.  It needs to be checked always, whenever a user is required so it needs to be added to require_user whenever the user is logged in.